Accordion Functionality & Visualizations Pages

shredtechular opened the pull request:
**UPDATE** - This PR is ready to merge now.
~~**Please NOTE** - This pull request builds upon **ALL** previous pull requests made.  All pull requests should be merged in order.  If any merge conflicts do occur as pull requests are accepted, I can remedy.~~

This PR focuses on adding accordion functionality that can be seen on the Visualizations pages as well as the rest of the content for those pages.  Additional notes found below:
- [x] Converted the content for the Visualizations pages
- [x] Created an accordion include that can be used on any page (and still is mostly utilizing markdown for everything) by following the below directions:
      - in the yml frontmatter of a page, add an ``accordion:`` array key with the headers used for each accordion list item
      - in the markdown file add liquid capture for each accordion's list item content, i.e. - ``{% capture accordion_entry_XX %} accordion content {% endcapture %}`` where XX is the corresponding item number. 
      - below all accordion items add an include to the accordion partial ``{% include accordion.html %}``
- [ ] **IMPORTANT NOTE** - there were a couple of links that I added to exclude from the html proofer as they were links that don't work even on the current site.  Should we exclude these items one off or do we want to take another action with these?  The links excluded you can see in the .travis file in the ``--href-ignore`` parameter
- [ ] **IMPORTANT NOTE 2** - there is some difficulty with external objects timing out or causing issues on the visualizations state page.  I'm seeing the same behavior on the current site, so I'm pretty sure this is an external issue.

critzo commented:
The two broken links should be:

- /tools/ndt
- http://dominichamon.com/blog/2012/12/visualizing-m-lab-data-with-bigquery-part-two/ 

The load times on the visualizations or timing out is an issue that we can ignore for now.

After updating the links and .travis.yml, LGTM @shredtechular

shredtechular commented:
Thanks @critzo for the info on that!  I updated those links in the code and then removed the ignores in the .travis.yml file as well.  Also rebased from origin/master so should be cleaner to look now!

critzo commented on _pages/02-visualizations.md:
```diff
@@ -2,8 +2,75 @@
+
+<p><iframe class="customIframe" src="https://docs.google.com/file/d/0B0dPEIVgbGveU1JjcnU3LXNNYXM/preview" name="customIframe_0" width="800" height="400" frameborder="0" marginwidth="800" marginheight="400"></iframe></p>
+
+If you're inspired to learn more, and maybe want to create your own, go to:  [https://dmadev.com/2012/12/10/visualizing-m-lab-data-with-bigquery-part-two/](https://dmadev.com/2012/12/10/visualizing-m-lab-data-with-bigquery-part-two/).
```
@shredtechular If you could update this link we should be good to go with this PR.
http://dominichamon.com/blog/2012/12/visualizing-m-lab-data-with-bigquery-part-two/

shredtechular commented:
Nice catch @critzo , that link has also now been updated to point to the same blog post page.

critzo commented:
@shredtechular LGTM merging.

critzo commented:
@shredtechular LGTM. Merging.

 Chris Ritzo
 Senior Technologist, Open Technology Institute @ New America
 740 15th Street NW, Suite 900
 Washington, DC 20036
 Public Key:    0x085B309C


On Mon, Feb 15, 2016 at 9:29 AM, Andrew Newhouse <notifications@github.com>
wrote:

> Nice catch @critzo <https://github.com/critzo> , that link has also now
> been updated to point to the same blog post page.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/m-lab/m-lab.github.io/pull/19#issuecomment-184228647>.
>
